src/app/lib/utilities/variantHandler/index.js

```diff
@@ -0,0 +1,21 @@
+const servicesWithVariants = {
+  serbian: ['lat', 'cyr'],
+  ukchina: ['simp', 'trad'],
+  zhongwen: ['simp', 'trad'],
+};
+
+// Remove leading slash from variant
+export const variantSanitzer = variant => variant && variant.replace('/', '');
```

Comment:
Typo here and in all places this is used:
```suggestion
export const variantSanitizer = variant => variant && variant.replace('/', '');
```

Reply:
Done :)